README.md

```diff
@@ -1,4 +1,4 @@
-# plotsbot
+# plotsbot [![Coverage Status](https://coveralls.io/repos/github/bassdeveloper/plotsbot/badge.svg?branch=coveralls)](https://coveralls.io/github/bassdeveloper/plotsbot?branch=coveralls)
```

Comment:
You don't need to add the badge just yet. Let's remove this for now and leave it for the FTO issue (we would need as much FTO material as we can so that we do not run out soon)